docs/cimplInstall.md

```diff
@@ -234,16 +250,18 @@ With [homebrew](#homebrew) installed, yarn can be installed by entering the foll
 ```
 brew install yarn
 ```
+**Close and re-open your command line tool** to make sure those settings take effect
 
 ### Git for macOS
 Git is a free and open source distributed version control system that allows developers working simultaneously on a single project to manage various updates to their code.
 
-Git is not a strict dependency (i.e., you can run the SHR-CLI tool without git), but it is recommended for CIMPL model developers to use git, along with a hosted git solution like [GitHub](https://github.com/) or [GitLab](https://gitlab.com/explore).
+Git is not a strict dependency (i.e., you can run SHR-CLI  without git), but it is recommended for CIMPL model developers to use git, along with a hosted git solution like [GitHub](https://github.com/) or [GitLab](https://gitlab.com/explore).
 
 With [homebrew](#homebrew) installed, git can be installed by entering the following in a terminal window:
 ```
 brew install git
 ```
+**Close and re-open your command line tool** to make sure those settings take effect 
```

Comment:
that's not necessary for Mac OS

Reply:
It is for Windows users, I can add that information to the note.  Nick suggested this in his review, and it would be helpful for Windows users.

Reply:
right so just put it in the Windows section or state, "for Windows users" if it's mixed in.